### PR TITLE
chore: modify values example

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
Update the examples in `values.yaml` about `notebooks.serverOptions` according to the changes here https://github.com/SwissDataScienceCenter/renku-notebooks/pull/213/files

### CODE REVIEW

File: charts/renku/values.yaml [hunk 1]
```diff
@@ -510,25 +510,31 @@ ui:
...
-  ## serverOptions allow for modifications to the notebook pod resource requests
+  ## serverOptions allow for modifications to the notebook pod resource requests and the UI rendering
   # serverOptions:
-  #   resources:
-  #     cpu_request:
-  #       displayName: Number of CPUs
-  #       type: integer
-  #       default: 1
-  #       range: [1, 4]
-  #     mem_request:
```
Comment (Panaetius) [inline charts/renku/values.yaml]: Is amount of memory not needed anymore?

Comment (lorenzo-cavazzi) [reply charts/renku/values.yaml]: Here I added just a few examples to show how the parameters are provided, not sure if I should copy the whole configuration from `notebooks` where the values are provided (non of these parameters are strictly needed for renku to work properly).

Comment (rokroskar) [reply charts/renku/values.yaml]: hmm but what is the harm in leaving the `mem_request` in there? To figure out how to change it, someone deploying the platform needs to look inside the notebooks service and they might not even be aware that it is possible to modify it.

**Comment by lorenzo-cavazzi** [comment]
@rokroskar true, I can copy all the code defined in "notebooks" if you think it's useful. I intended this as an example but maybe it can be confusing for users not to find here `mem_request` since it's defined somewhere else.

**Comment by lorenzo-cavazzi** [comment]
can't merge by myself since I'm unauthorized :slightly_frowning_face:

**Pull request merged**